tests/foreman/ui/test_syncplan.py

```diff
@@ -83,14 +39,29 @@ def test_positive_create_basic(self, test_data):
 
         """
         with Session(self.browser) as session:
-            make_syncplan(
-                session,
-                org=self.organization.name,
-                name=test_data['name'],
-                description=test_data['desc'],
-                sync_interval=test_data['interval'],
-            )
-            self.assertIsNotNone(self.syncplan.search(test_data['name']))
+            for value in generate_strings_list():
```

Comment:
IMHO this is totally redundant testing as I am testing name/desc in one test and sync interval in another test and all the data points are being tested.  But since you insist I can add another loop.